LINUX_INSTALL.md

```diff
@@ -2,18 +2,18 @@
 
 The minimum requirements are:
  - python 2.7 
- - glibc >= 2.12 (Centos >= 6, Ubuntu >= 10.10)
+ - glibc >= 2.11 (Centos >= 6, Ubuntu >= 10.04)
  - For Neural Network support, glibc >= 2.17 is needed. Centos >= 7 and Ubuntu >= 13.04
- - libstdc++ >= 6.0.21
+ - libstdc++ >= 6.0.19
  - libgconf-2-4 (on Ubuntu 17.10 and later)
 
 ## Ubuntu
 
 ### Ubuntu >= 17.10
```

Comment:
Do you think it would be a good idea to link to the relevant docker files?

Reply:
Probably not under LINUX_INSTALL (the Docker files aren't really relevant to installation) but under BUILD I will.